pkg/trace/config/config.go

```diff
@@ -289,12 +289,17 @@ type AgentConfig struct {
 	ExtraAggregators []string
 
 	// Sampler configuration
-	ExtraSampleRate    float64
-	TargetTPS          float64
-	ErrorTPS           float64
-	DisableRareSampler bool
-	MaxEPS             float64
-	MaxRemoteTPS       float64
+	ExtraSampleRate float64
+	TargetTPS       float64
+	ErrorTPS        float64
+	MaxEPS          float64
+	MaxRemoteTPS    float64
+
+	// Rare Sampler configuation
+	DisableRareSampler        bool
```

Comment:
```suggestion
	RareSamplerDisabled        bool
```
Just to be on par with all the others :)